.gitmodules

```diff
@@ -58,3 +55,6 @@
 [submodule "openpain-BrainNetworkChange_Mano"]
 	path = projects/openpain/BrainNetworkChange_Mano
 	url = https://github.com/big-data-lab-team/openpain-BrainNetworkChange_Mano.git
+[submodule "projects/PERFORM_Dataset__one_control_subject"]
+  path = projects/PERFORM_Dataset__one_control_subject
+  url = https://github.com/conp-bot/conp-dataset-PERFORM_Dataset__one_control_subject.git
```

Comment:
not sure why this has moved @joeyzhou98, might be worth checking for the next PRs.